source/elements/oneDAL/source/glossary.rst

```diff
@@ -0,0 +1,159 @@
+.. _glossary:
+
+=========
+Glossary
+=========
+
+Machine learning terms
+======================
+
+.. glossary::
+    :sorted:
+
+    Categorical feature
+        A :term:`feature` with a discrete set of possible values without ordering.
```

Comment:
I think it's finite, not discrete

Reply:
I meant (discrete data)[https://stats.stackexchange.com/questions/206/what-is-the-difference-between-discrete-data-and-continuous-data]. Maybe, it is better to reformulate it like this?
"A feature with a discrete domain without ordering between values"

Reply:
'discrete, continuous, ordinal, categorical' (and Mikhail A also suggested interval) were aligned with http://dmg.org/pmml/v4-1/DataDictionary.html in the lib